_posts/2021-01-06-introducing-artsy-engineering-radio.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: epic
+title: "Introducing Artsy Engineering Radio"
+date: 2021-01-06
+categories: [podcast, community, learning]
+author: [steve-hicks, jonallured]
+---
+
+Available now [on iTunes][itunes] and hopefully your preferred podcast provider 🤞 is Artsy Engineering Radio!
+We'll release new episodes every other week. Our target episode length is 30 minutes. Some episodes will be
+technical and others less so. Because [we work in the open][open-source-by-default] it's easy for us to talk about
+our work in public.
+
+Solving problems in the world of software engineering can mean a lot of different things, and this podcast will
```

Comment:
suggestion: move this paragraph above the previous paragraph. this feels much more like an intro!

Reply:
oop I realize that this is a very unclear comment; I thought there was a paragraph break after "artsy engineering radio!" and there is not. Here's what I was envisioning:

```
Available now on iTunes and hopefully your preferred podcast provider 🤞 is Artsy Engineering Radio! 

Solving problems in the world of software engineering can mean a lot of different things, and this podcast will explore what that looks like at Artsy. If you've followed our blog, you can expect a podcast that sounds like it. There are a ton of amazing engineers here at Artsy and we're excited for you to hear their voices and stories. Our hope is that this podcast will reduce the friction and get more contributions shared with the world.

We'll release new episodes every other week. Our target episode length is 30 minutes. Some episodes will be technical and others less so. Because we work in the open it's easy for us to talk about our work in public.
```

Reply:
I like this, I will make this change.